zerver/context_processors.py

```diff
@@ -152,13 +152,29 @@ def login_context(request: HttpRequest) -> Dict[str, Any]:
         realm_description = get_realm_rendered_description(realm)
         realm_invite_required = realm.invite_required
 
+    def get_apple_locale() -> str:
+        supported_locales = ['ar_SA', 'ca_ES', 'cs_CZ', 'da_DK', 'de_DE',
+                             'en_US', 'es_ES', 'es_MX', 'fi_FI', 'fr_FR',
+                             'hr_HR', 'hu_HU', 'el_GR', 'nl_NL', 'no_NO',
+                             'it_IT', 'iw_IL', 'ja_JP', 'ko_KR', 'ms_MY',
+                             'pt_PT', 'ro_RO', 'ru_RU', 'sk_SK', 'sv_SE',
+                             'pl_PL', 'th_TH', 'tr_TR', 'uk_UA', 'vi_VI'
+                             'zh_CN', 'zh_HK', 'id_ID', 'pt_BR']
```

Comment:
Missing comma gives this the wrong meaning -- there's an element `'vi_VIzh_CN'` :stuck_out_tongue: